src/helpers/observer.js

```diff
@@ -16,6 +16,22 @@ const TOLERANCE_PX = 10;
 const {scrollIfNeeded, resetScrolling} = makeScroller();
 let next;
 
+function findScrollableContainers(element) {
+    if (!element) {
+        return [];
+    }
+    const scrollableContainers = [];
+    let parent = element;
+    while (parent) {
+        const {overflow} = window.getComputedStyle(parent);
+        if (overflow.split(" ").some(o => o === "auto" || o === "scroll")) {
```

Comment:
can "overflow" ever be undefined (on any browser etc)?

Reply:
not according to the spec - it's always a defined string value and defaults to `auto`